presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanNodeStatsSummarizer.java

```diff
@@ -154,6 +154,10 @@ private PlanNodeStatsSummarizer() {}
         List<PlanNodeStats> stats = new ArrayList<>();
         for (Map.Entry<PlanNodeId, Long> entry : planNodeWallMillis.entrySet()) {
             PlanNodeId planNodeId = entry.getKey();
+            if (!planNodeInputPositions.containsKey(planNodeId)) {
+                // We should not fail the query if there is no input stats.
```

Comment:
This comment is not required, and could get out of date. Let's save comments for thornier pieces of code.